packages/frontend/src/hooks/dbtCloud/useProjectDbtCloudSettings.ts

```diff
@@ -34,6 +34,44 @@ const post = async (projectUuid: string, data: CreateDbtCloudIntegration) =>
         body: JSON.stringify(data),
     });
 
+const doDelete = async (projectUuid: string) =>
+    lightdashApi<undefined>({
+        url: `/projects/${projectUuid}/integrations/dbt-cloud/settings`,
+        method: 'DELETE',
+        body: undefined,
+    });
+
+export const useProjectDbtCloudDeleteMutation = (projectUuid: string) => {
+    const queryClient = useQueryClient();
+    const { showToastSuccess, showToastError } = useToaster();
+    if (projectUuid === undefined) {
+        throw new Error(
+            'Must use useProjectDbtCloudDeleteMutation hook under react-router path with projectUuid available',
+        );
+    }
+    return useMutation<undefined, ApiError, undefined>(
+        () => doDelete(projectUuid),
+        {
+            mutationKey: ['updated-dbt-cloud', projectUuid],
```

Comment:
does mutations require to have a mutationKey? I am just curious what problem it solves

Reply:
If the mutation returns a result it's stored in the key, that way it caches the returned object without having to mutate again. In this case it's pointless because the delete api returns nothing.